lomap/algorithms/optimal_run.py

```diff
@@ -18,7 +18,7 @@
 import traceback
 import logging
 
-import pp
```

Comment:
Don't comment out `pp` python package. Instead test if the import is successful or not, and save it to a flag. Use the flag to guard the code dependent on `pp`.